### PR TITLE
Code Samples demonstrating Audio Profiles

### PR DESCRIPTION
**Pull request by CallistoCF**
- [x] TODO: add link to canonical samples
 - [x] TODO: list out the region tags this adds

### CODE REVIEW

**Comment by beccca** [comment]
This repo doesn't have CLA labels – /cc @crwilcox ???

File: samples/audioProfile.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/*
+* TODO(developer): Uncomment the following lines before running the sample.
```
Comment (beccca) [inline samples/audioProfile.js]: What is this comment here for?

File: samples/audioProfile.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+function synthesizeText(
+  text,
+  outputFile,
+  effectsProfileId,
+  languageCode,
+  ssmlGender
+) {
+  //[START tts_synthesizeText]
+  //Synthesizes speech from the input string of text
```
Comment (beccca) [inline samples/audioProfile.js]: remove

File: samples/audioProfile.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+  //Imports the Google Cloud client library
```
Comment (beccca) [inline samples/audioProfile.js]: `// Imports the Google Cloud client library`

File: samples/audioProfile.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+  const speech = require('@google-cloud/text-to-speech');
+  const fs = require('fs');
+
+  //Creates a client
```
Comment (beccca) [inline samples/audioProfile.js]: `// Creates a client`

Fix whitespace

File: samples/audioProfile.js (hunk 3 above)
Comment (beccca) [reply samples/audioProfile.js]: Fix whitespace

File: samples/system-test/audioProfile.test.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+const tools = require(`@google-cloud/nodejs-repo-tools`);
+
+const cmd = `node audioProfile.js`;
+const cwd = path.join(__dirname, `..`);
+const text = `Hello Everybody!  This is an Audio Profile Optimized Sound Byte.  `;
+const outputFile1 = `phonetest.mp3`;
+const outputFile2 = `homeTheatreTest.mp3`;
+const outputFile3 = `carAudioTest.mp3`;
+const outputFile4 = `watchAudioTest.mp3`;
+// const files = [`hello.txt`, `hello.ssml`].map(name => {
```
Comment (beccca) [inline samples/system-test/audioProfile.test.js]: What is this commented out code? Remove

File: samples/audioProfile.js [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+//const outputFile = 'Local path to save audio file to, e.g. output.mp3';
+
+function synthesizeText(
+  text,
+  outputFile,
+  effectsProfileId,
+  languageCode,
+  ssmlGender
+) {
+  //[START tts_synthesizeText]
```
Comment (beccca) [inline samples/audioProfile.js]: fix region tag

File: samples/audioProfile.js [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+    fs.writeFile(outputFile, response.audioContent, 'binary', err => {
+      if (err) {
+        console.error('ERROR:', err);
+        return;
+      }
+      console.log(`Audio content written to file: ${outputFile}`);
+    });
+  });
+  // [END tts_synthesizeText]
```
Comment (beccca) [inline samples/audioProfile.js]: fix region tag

File: samples/system-test/audioProfile.test.js [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+'use strict';
+
+const fs = require(`fs`);
+const path = require(`path`);
+const test = require(`ava`);
+const tools = require(`@google-cloud/nodejs-repo-tools`);
+
+const cmd = `node audioProfile.js`;
+const cwd = path.join(__dirname, `..`);
+const text = `Hello Everybody!  This is an Audio Profile Optimized Sound Byte.  `;
```
Comment (beccca) [inline samples/system-test/audioProfile.test.js]: extraneous whitespace padding right of this string

**Comment by CallistoCF** [comment]
Cannonicals: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/texttospeech/cloud-client/audio_profile.py

Region Tags:  tts_synthesize_text_with_audio_profile

File: samples/audioProfile.js [hunk 9]
```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2017, Google, Inc.
```
Comment (happyhuman) [inline samples/audioProfile.js]: 2018, LLC instead of Inc.

File: samples/system-test/audioProfile.test.js [hunk 10]
```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2018, Google, Inc.
```
Comment (happyhuman) [inline samples/system-test/audioProfile.test.js]: LLC

**Comment by beccca** [comment]
This has been approved. I'm off of PRs for a few weeks, unassigning myself from this

/fyi @CallistoCF and @happyhuman

**Pull request merged**